actionpack/lib/action_controller/metal/helpers.rb

```diff
@@ -12,6 +12,12 @@ module ActionController
   # In previous versions of \Rails the controller will include a helper which
   # matches the name of the controller, e.g., <tt>MyController</tt> will automatically
   # include <tt>MyHelper</tt>. To return old behavior set +config.action_controller.include_all_helpers+ to +false+.
```

Comment:
"To return old behavior" is ungrammatical, can you fix that while you're editing this paragraph?

Reply:
will do!